src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -24,7 +28,8 @@ private List<Product> getAllProductsFromShoppingCart(Long userId) {
         Product macBook = new Product("MacBook Air 2020", BigDecimal.valueOf(1399));
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
-        // TODO: add log message about successful fetched data from DB
+        logger.info("Successfully added product do data base: {}",iphone,macBook,xiaomi);
```

Comment:
The logger message 'Successfully added product do data base' is misleading because there is no database operation here. Moreover, the logger should log each product separately for clarity and avoid passing multiple arguments which are not being used in the message format. Correct the message to reflect what the method actually does.

Reply:
ok, done